<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PlayerDialog</class>
 <widget class="QDialog" name="PlayerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>700</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>700</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>PlayerDialog</string>
  </property>
  <widget class="QPushButton" name="m_openPB">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>485</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>打开</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>700</width>
     <height>450</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <widget class="QLabel" name="m_showLabel">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>700</width>
      <height>450</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="m_curtTimeLabel">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:00:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QSlider" name="m_progressSlider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>460</y>
     <width>500</width>
     <height>18</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="m_totalTimeLabel">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>460</y>
     <width>80</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>11</pointsize>
    </font>
   </property>
   <property name="text">
    <string>00:03:00</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="m_resumePB">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>485</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>恢复</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_pausePB">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>485</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>暂停</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_stopPB">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>485</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>停止</string>
   </property>
  </widget>
  <widget class="QPushButton" name="m_setUrlPB">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>485</y>
     <width>70</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>url</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
